src/components/App/App.module.scss

```diff
@@ -1,3 +1,9 @@
-.div {
-  background-color: aqua;
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.forFooterWrapper {
```

Comment:
I suggest to follow kebab-case naming for classes, it's more common to see this style :)
Also check this out https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Naming